crates/pyrefly_python/src/ignore.rs

```diff
@@ -306,6 +315,8 @@ mod tests {
 
     #[test]
     fn test_parse_ignores() {
+        // TODO: The second component of `expect` (the error code)
```

Comment:
could you add a test for the difference in only_all true vs false? (I'm still confused as to what it does)